libs/journeys/ui/src/components/TemplateView/CreateJourneyButton/CreateJourneyButton.spec.tsx

```diff
@@ -298,7 +298,35 @@ describe('CreateJourneyButton', () => {
     )
 
     await waitFor(() =>
-      expect(screen.getByTestId('CopyToTeamDialog')).toBeInTheDocument()
+      expect(queryByTestId('CopyToTeamDialog')).not.toBeInTheDocument()
```

Comment:
```suggestion
      expect(screen.queryByTestId('CopyToTeamDialog')).not.toBeInTheDocument()
```
use `screen` rather than assigning render to `const {queryByTestId}`